src/org/opendatakit/briefcase/ui/SettingsPanel.java

```diff
@@ -83,14 +85,20 @@ public SettingsPanel(MainBriefcaseWindow parentWindow) {
         chkParallel.setSelected(BriefcasePreferences.getBriefcaseParallelPullsProperty());
         chkParallel.addActionListener(new ParallelPullToggleListener());
 
+        lblTrackingConsent = new JLabel(MessageStrings.TRACKING_CONSENT);
```

Comment:
I would have used this constructor
https://docs.oracle.com/javase/7/docs/api/javax/swing/JCheckBox.html#JCheckBox(java.lang.String)
to provide the label. Then clicking on the label would toggle the checkbox. You did the right thing, probably, by continuing in the existing pattern.